specs/adagrams_spec.rb

```diff
@@ -0,0 +1,173 @@
+require 'minitest/autorun'
+require 'minitest/reporters'
+require 'minitest/skip_dsl'
+
+require_relative '../lib/adagrams'
+
+# Get that nice colorized output
+Minitest::Reporters.use! Minitest::Reporters::SpecReporter.new
+
+describe 'Adagrams' do
+  describe 'draw_letters method' do
+    it 'draws ten letters from the letter pool' do
+      drawn_letters = draw_letters
```

Comment:
I'm imagining a test where we run it 100 times, for two reasons:

- Make sure drawing doesn't reduce the pool (would fail after 10)
- Make sure none of the drawn hands contains too many copies of a letter

This might just be overkill though.

Reply:
I think this is overkill! I think it would be fine if we skated over this part -- I think that testing that something is a length of 10 is already the level of detail we need on wave 1